src/utils/server/props/typeObjToValues.ts

```diff
@@ -0,0 +1,28 @@
+import { convertInternalTypeToUnderstandableTypes, type TsTypeDescriptor } from '@utils/server';
+
+export const typeObjToValues = (
```

Comment:
Heuristic conversion is performed because clean values cannot be obtained by `react-docgen` extraction.